### PR TITLE
add one scenario more

### PR DESCRIPTION
**Pull request by Cxristian**

### CODE REVIEW

File: src/main/java/core/utils/DriverMethods.java [hunk 1]
```diff
@@ -27,8 +28,9 @@
  */
 public final class DriverMethods {
 
-    protected WebDriverWait webDriverWait;
-    protected WebDriver webDriver;
+    private static WebDriverWait webDriverWait = WebDriverManager.getInstance().getWait();;
```
Comment (LimbertVargas) [inline src/main/java/core/utils/DriverMethods.java]: The last ";" symbol is over

File: src/test/resources/features/Account.feature [hunk 2]
```diff
@@ -41,3 +41,12 @@ Feature: Manage Account
...
       |admin member |admin.test3@outlook.com |Owner  |
+
+  Scenario: Delete a account in Pivotal Tracker
+    When I go to the Account Page
+    And I create a new account "New Account"
+    Then I should see the new account in Account Plans Page
+    When I go to the Settings tab inside Account page
+    And I delete the account that was created
+    Then I should see a yellow message "New Account was successfully deleted." in Accounts Page
+    And I should see all of the accounts except the deleted account
```
Comment (LimbertVargas) [inline src/test/resources/features/Account.feature]: please add an additional line

File: src/test/java/steps/AccountSteps.java [hunk 3]
```diff
@@ -132,4 +135,25 @@ public void verifyTheMemberThatWasAddedInTheTableOfTheMemberPageOfAccount() {
...
+    public void verifyShouldSeeAYellowMessageInAccountsPage(final String message) {
+        Assert.assertEquals(message, accountPage.getMessageDelete(), "The message is not the correct");
+    }
+
+    @And("I should see all of the accounts except the deleted account")
+    public void verifyShouldSeeAllOfTheAccountsExceptTheDeletedAccount() {
+        Assert.assertTrue(accountPage.elementDisappear(account.getNameAccount()).isEmpty());
+    }
+
+//    @When("I delete the member {string} from Pivotal Tracker account in the Account Memberships")
```
Comment (titor-oopart) [inline src/test/java/steps/AccountSteps.java]: what is this???

File: src/test/resources/features/Account.feature [hunk 4]
```diff
@@ -41,3 +41,25 @@ Feature: Manage Account
...
+
+#  @deleteAccount
```
Comment (titor-oopart) [inline src/test/resources/features/Account.feature]: Delete this scenario

**Review by titor-oopart — APPROVED** [review]

**Pull request merged**